server/portal/apps/accounts/migrations/0005_auto_20210312_1730.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.17 on 2021-03-12 17:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0004_auto_20200318_2055'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='notificationpreferences',
+            name='announcements',
+            field=models.BooleanField(default=True, verbose_name='Receive occasional announcements from CEP'),
```

Comment:
Any way we can avoid this line from being a committed migration? Could we instead ignore this field in the `makemigrations`, and have every portal generate this migration as a part of the portal setup? For background, the `CEP` string comes from the variable in the model that will differ depending on the portal

Reply:
Would it be easier to tweak the migration itself to format the string using `'Receive occasional announcements from {}'.format(settings.PORTAL_NAMESPACE))`?

Reply:
Having custom migrations per portal can lead to headaches when updating a model in the future, especially if we're checking in some migrations and not others. Are we trying to keep the flexibility of a custom verbose_name?  If so, I think there are better ways to do that, like overriding the value in the form.

Reply:
> Would it be easier to tweak the migration itself to format the string using `'Receive occasional announcements from {}'.format(settings.PORTAL_NAMESPACE))`?

If we can't override the return value without affecting the migration, I think this would be a better option than not checking in a migration at all.

Reply:
It looks like this model/field isn't currently being used, and we may be able to just modify the verbose_name attr to use generic text. If a specific portal needs to customize the text once notifications are implemented then the place for that may be a downstream customization

Reply:
Good with me. Thanks!